sdk/search/Azure.Search.Documents/tests/Samples/FieldBuilder/AnalyzerAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Search.Documents.Indexes.Models;
+
+namespace Azure.Search.Documents.Tests.Samples
```

Comment:
I'd rather not because the final API will be different. Either way they'd have to port the code. I want it clear this is a sample, though maybe we should remove ".Tests" from the namespace so it's just `Azure.Search.Documents.Samples` - same for `IndexingParametersExtensions`. Cool?

Reply:
Done